### DISCUSSION TITLE
Quality of life bracket editor improvement in tournament client

### DISCUSSION THREAD

**Discussion by Kuina-chan** [ppy/osu#24658] [Ideas] [answered]
I suggest a way to choose multiple match to move around (while holding Ctrl for example) and allow an option to choose that whole selected matches to be loser bracket or not.
Making Ctrl+S to save changes rather than switching to schedule tab.
Maybe a preset for standard tournament format (double elimination style from RO16)

**Comment by peppy** [comment, marked as answer]
Selecting multiple matches is going to take a bit more effort, but I've added ctrl+s support via https://github.com/ppy/osu/pull/24679.
